install/kubernetes/cilium/values.yaml.tmpl

```diff
@@ -4,6 +4,8 @@
 # For example: 1.7, 1.8, 1.9
 # upgradeCompatibility: '1.8'
 
+namespaceOverride: ""
```

Comment:
Can you please add a description for this value? (Using the `# --` comment syntax as for the other values below). It would be rendered in Documentation/helm-values.rst and install/kubernetes/cilium/README.md and would help understand what the value stands for.

Reply:
Added comment docs in values and generated docs for readme, helm and values template